src/owasp-dependency-check-false-positives.xml

```diff
@@ -158,4 +158,13 @@
     <sha1>1a754a5dd672218a2ac667d7ff2b28df7a5a240e</sha1>
     <cve>CVE-2022-25647</cve>
   </suppress>
+
+  <!-- 9.4.x is not affected https://github.com/eclipse/jetty.project/issues/8161#issuecomment-1178728623-->
+  <suppress>
```

Comment:
We can revert this change since upstream has fixed the false positive https://github.com/eclipse/jetty.project/issues/8161.

Reply:
My latest OWASP check gave green here: https://github.com/apache/pulsar/runs/7294406368?check_suite_focus=true

So I wonder even whether we should bump jetty version, while bump to new version is always a valid improvement from my side.

Reply:
Our jetty version is still vulnerable to https://github.com/eclipse/jetty.project/security/advisories/GHSA-cj7v-27pg-wf7q 
so it's better to upgrade